feat: LEAP-355: Make relations work with TimeTraveler

robot-ci-heartex opened the pull request:
Hi @Gondragos!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7095981351) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1636

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7095981351

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7169093260

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7169093260

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7169228178

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7169430131

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7171605384

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7263280842

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7265909390

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7265965249

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7268934890

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7268934890

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7268960930

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7269170651

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7288862564

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7335729477

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

Gondragos committed:
Merge branch 'develop' into fb-leap-355/relations-improvements

# Conflicts:
#	label_studio/frontend/dist/lsf/js/main.js
#	label_studio/frontend/dist/lsf/js/main.js.map
#	label_studio/frontend/dist/lsf/version.json

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7452644607

hlomzik commented:
/frontend build lsf master
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7454652347)
> **Error**: failed to get build

Commit:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458072331

Commit:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458072331

Commit:
Merge branch 'origin/develop' into 'fb-leap-355/relations-improvements'

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458100224

nikitabelonogov committed:
Merge remote-tracking branch 'origin/develop' into fb-leap-355/relations-improvements

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7462470608
